### ISSUE TITLE
Library: Suggest <dialog> tag for form

### ISSUE THREAD

**Issue by MaoShizhong** [TheOdinProject/curriculum#26094]
### Describe your suggestion

Was thinking based on something said in the general chat in Discord - in step 4 of the library project, would it be beneficial to add a small suggestion for "bring up a form" upon "add new book" for the `<dialog>` tag?

Something like
"How you decide to display this is up to you. One such approach may be via a modal using the `<dialog>` tag." 
perhaps with a link to the MDN docs on that tag?

Or perhaps a more open-ended suggestion such as
"How you decide... ...For example, you may wish to have a form appear in a sidebar or you may wish to explore the `<dialog>` tag."

Reason is a modal is a popular way to do this and also a common front-end feature. `<dialog>` makes this very easy to implement as it has styles and behaviours implemented by default, meaning no need to manually create modals as *many* online tutorials try to teach (simply googling "how to create a modal/popup in JS/HTML" etc. often leads to older tutorials that go through finicky manual methods that can now be achieved with the native dialog tag).

I believe it to both be a useful thing for people to at least be made aware of (I still see many people trying to implement manual modals, not knowing that `<dialog>` will make their lives much easier!) both for the project but mainly because it's a feature many might wish to use as part of future projects.

Thoughts?

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-javascript-library

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### (Optional) Discord Name

Mao

### (Optional) Additional Comments

Discord username/identifier: mao_sz

**Comment by codyloyd** [comment]
I agree.  The dialog tag is dope... it's relatively new but is now supported in all major browsers, so users should probably get comfortable with it. (it's easy, anyway)

I'll move this to 'in progress' and assign you @MaoShizhong

**Comment by MaoShizhong** [comment]
> I agree. The dialog tag is dope... it's relatively new but is now supported in all major browsers, so users should probably get comfortable with it. (it's easy, anyway)
> 
> I'll move this to 'in progress' and assign you @MaoShizhong

Not done an issue specifically before but just confirming this means I can submit a PR for this?
